agir/front/components/genericComponents/EventCard.js

```diff
@@ -250,6 +250,14 @@ const EventCard = (props) => {
     compteRendu,
   } = props;
   const history = useHistory();
+
+  const now = DateTime.local();
+  const pending = now >= schedule.start && now <= schedule.end;
+
+  const eventDate = pending
+    ? displayIntervalEnd(schedule)
+    : displayIntervalStart(schedule);
```

Comment:
Pas besoin de recalculer `eventDate` à chaque render :
```suggestion
    const eventDate = React.useMemo(() => {
       const now = DateTime.local();
	    const pending = now >= schedule.start && now <= schedule.end;
	  
	    return pending
	      ? displayIntervalEnd(schedule)
	      : displayIntervalStart(schedule);
	}, [schedule])
```

Reply:
Mais à chaque render la valeur de 'now' sera différente et donc fera différer 'pending' non ?

Reply:
Oui, effectivement tu peux aussi garder la déclaration de `now` en dehors du hook pour que la date se mette à jour automatiquement à chaque render, même si ça ne sera pas forcément très compréhensible pour l'utilisateur, vu que la date changera de manière aléatoire selon si le composant se met à jour ou pas.

Reply:
Et si la valeur de `now` change à chaque render il n'y aura effectivemnt plus besoin de mettre tout ça dans un `useMemo`. Je te laisse voir, mais sinon c'est ok pour moi.